org/views.py

```diff
@@ -190,43 +188,80 @@ def AddVolunteer(request, org_id):
-    method='PUT',
-    request_body=EditOrgSerializer,
-    responses={
-        '200': set_example(responses.update_org_200),
-        '400': set_example(responses.org_not_present_400),
-        '401': set_example(responses.user_unauthorized_401),
-        '403': set_example(responses.admin_access_403),
-    }
-)
-@api_view(['PUT'])
-@permission_classes([IsAuthenticated])
-def EditOrg(request, org_id):
-    user = request.user
-    try:
-        org = Org.objects.get(pk=org_id)
-    except Org.DoesNotExist:
-        return Response(responses.org_not_present_400, status.HTTP_400_BAD_REQUEST)
+class EditOrgView(APIView):
+    permission_classes = [IsAuthenticated]
 
-    try:
-        member = Member.objects.get(user=user, org=org)
-    except Member.DoesNotExist:
-        return Response(responses.admin_access_403, status.HTTP_403_FORBIDDEN)
+    @swagger_auto_schema(
+        operation_id="edit_org",
+        request_body=EditOrgSerializer,
+        responses={
+            '200': set_example(responses.update_org_200),
+            '400': set_example(responses.org_not_present_400),
+            '401': set_example(responses.user_unauthorized_401),
+            '403': set_example(responses.admin_access_403),
+        }
+    )
+    def put(self, request, org_id):
+        user = request.user
+        try:
+            org = Org.objects.get(pk=org_id)
+        except Org.DoesNotExist:
+            return Response(responses.org_not_present_400, status.HTTP_400_BAD_REQUEST)
+
+        try:
+            member = Member.objects.get(user=user, org=org)
+        except Member.DoesNotExist:
+            return Response(responses.admin_access_403, status.HTTP_403_FORBIDDEN)
 
-    isadmin = member.group.perm_obj.permissions_to_integer()
-    # Checking if the isadmin is odd or even, if odd then the IS_ADMIN permission is enabled for the user
-    if isadmin % 2 == 1:
-        if request.method == "PUT":
+        if member.group.perm_obj.permissions[Permissions.IS_ADMIN]:
             serializer = EditOrgSerializer(org, data=request.data)
             if serializer.is_valid():
                 serializer.save()
                 return Response(responses.update_org_200, status.HTTP_200_OK)
             return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
-    else:
-        return Response(responses.admin_access_403, status.HTTP_403_FORBIDDEN)
+        else:
+            return Response(responses.admin_access_403, status.HTTP_403_FORBIDDEN)
+
+    @swagger_auto_schema(
+        operation_id="get_org",
+        responses={
+            '200': set_example([{
+                "id": 1,
+                "route_slug": "slug",
+                "can_join_without_invite": True,
+                "name": "test",
+                "tagline": "test",
+                "about": "test",
+                "profile_pic":  "null",
+                "cover_pic":  "null"
+            }]),
```

Comment:
```suggestion
            '200': set_example(responses.get_org_200),
```
Please extract it into responses.py